icu4c/APIChangeReport.md

```diff
@@ -0,0 +1,1094 @@
+| uchar.h | const USet* u_getBinaryPropertySet(UProperty, UErrorCode*) |  Draft→StableICU 63
+| ucpmap.h | UChar32 ucpmap_getRange(const UCPMap*, UChar32, UCPMapRangeOption, uint32_t, UCPMapValueFilter*, const void*, uint32_t*) |  Draft→StableICU 63
+| ucpmap.h | <tt>enum</tt> UCPMapRangeOption::UCPMAP_RANGE_FIXED_ALL_SURROGATES |  Draft→StableICU 63
+| ucpmap.h | <tt>enum</tt> UCPMapRangeOption::UCPMAP_RANGE_FIXED_LEAD_SURROGATES |  Draft→StableICU 63
+| ucpmap.h | <tt>enum</tt> UCPMapRangeOption::UCPMAP_RANGE_NORMAL |  Draft→StableICU 63
+| ucpmap.h | uint32_t ucpmap_get(const UCPMap*, UChar32) |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_16 |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_32 |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_8 |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_ASCII_GET |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_FAST_BMP_GET |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_FAST_GET |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_FAST_SUPP_GET |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_FAST_U16_NEXT |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_FAST_U16_PREV |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_FAST_U8_NEXT |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_FAST_U8_PREV |  Draft→StableICU 63
+| ucptrie.h | <tt>#define</tt> UCPTRIE_SMALL_GET |  Draft→StableICU 63
+| ucptrie.h | UCPTrie* ucptrie_openFromBinary(UCPTrieType, UCPTrieValueWidth, const void*, int32_t, int32_t*, UErrorCode*) |  Draft→StableICU 63
+| ucptrie.h | UCPTrieType ucptrie_getType(const UCPTrie*) |  Draft→StableICU 63
+| ucptrie.h | UCPTrieValueWidth ucptrie_getValueWidth(const UCPTrie*) |  Draft→StableICU 63
+| ucptrie.h | UChar32 ucptrie_getRange(const UCPTrie*, UChar32, UCPMapRangeOption, uint32_t, UCPMapValueFilter*, const void*, uint32_t*) |  Draft→StableICU 63
+| ucptrie.h | <tt>enum</tt> UCPTrieType::UCPTRIE_TYPE_ANY |  Draft→StableICU 63
+| ucptrie.h | <tt>enum</tt> UCPTrieType::UCPTRIE_TYPE_FAST |  Draft→StableICU 63
+| ucptrie.h | <tt>enum</tt> UCPTrieType::UCPTRIE_TYPE_SMALL |  Draft→StableICU 63
+| ucptrie.h | <tt>enum</tt> UCPTrieValueWidth::UCPTRIE_VALUE_BITS_16 |  Draft→StableICU 63
+| ucptrie.h | <tt>enum</tt> UCPTrieValueWidth::UCPTRIE_VALUE_BITS_32 |  Draft→StableICU 63
+| ucptrie.h | <tt>enum</tt> UCPTrieValueWidth::UCPTRIE_VALUE_BITS_8 |  Draft→StableICU 63
+| ucptrie.h | <tt>enum</tt> UCPTrieValueWidth::UCPTRIE_VALUE_BITS_ANY |  Draft→StableICU 63
+| ucptrie.h | int32_t ucptrie_toBinary(const UCPTrie*, void*, int32_t, UErrorCode*) |  Draft→StableICU 63
+| ucptrie.h | uint32_t ucptrie_get(const UCPTrie*, UChar32) |  Draft→StableICU 63
+| ucptrie.h | void ucptrie_close(UCPTrie*) |  Draft→StableICU 63
+| ulistformatter.h | <tt>enum</tt> UListFormatterField::ULISTFMT_ELEMENT_FIELD |  Draft→StableICU 63
```

Comment:
Shane says should be draft until 67

Reply:
yeah, i had fixed the enum but kept all members stable. fixed